tests/jballerina-unit-test/src/test/resources/test-src/imports/OverriddenPredeclaredImportsTestProject/overridden-predeclared-modules.bal

```diff
@@ -54,9 +55,14 @@ function testStartsWithFunctionInString() returns boolean {
     return 'strings:startsWith(str, "Hello");
 }
 
+function testCallInFunction()returns any|error {
```

Comment:
```suggestion
function testCallInFunction() returns any|error {
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/36700